src/modlunky2/ui/trackers/runstate.py

```diff
@@ -648,40 +588,36 @@ def update(self, game_state: State):
         if player is None:
             return
 
-        inventory = player.inventory
         state = player.state
         last_state = player.last_state
```

Comment:
can these be removed in favor of just passing `player.state` instead of `state`? `state` is overloaded so would nice to use the `player.state` directly now

Reply:
yeah, I suspect I forgot `last_state` is just a player field, not something `RunState` needs to track